src/components/templates/order-table/use-order-column.tsx

```diff
@@ -64,59 +61,56 @@ const useOrderTableColums = () => {
               }}
               color={getColor(row.index)}
             />
-          </Table.Cell>
         ),
       },
       {
         Header: "Fulfillment",
         accessor: "fulfillment_status",
-        Cell: ({ cell: { value }, index }) => (
-          <Table.Cell key={index}>{value}</Table.Cell>
+        Cell: ({ cell: { value } }) => (
+          {value}
```

Comment:
invalid syntax, wrap in div.

```suggestion
        Cell: ({ cell: { value } }) => (
          <div>{value}</div>
```